magellan-library/src/test/java/com/wealthfront/magellan/coroutines/CreatedLifecycleScopeTest.kt

```diff
@@ -41,86 +38,77 @@ internal class CreatedLifecycleScopeTest {
   }
 
   @Test
-  fun cancelBeforeCreated() {
-    runBlockingTest {
+  fun cancelBeforeCreated() = runTest {
+    val async = createdScope.async { delay(5000) }
+    assertThat(async.isCancelled).isTrue()
+    assertThat(async.getCancellationException()).hasMessageThat().contains("Not created yet")
```

Comment:
there is probly a better way to inspect these versus using `InternalCoroutinesApi` (try-catch block?). But leaving it to keep this PR simple as possible